### PR TITLE
fix(api): monitor the pressure sensor more

### PR DESCRIPTION
**Pull request by ryanthecoder**
# Overview
This will allow us to monitor the pressure sensor more, specifically it should help us catch and differentiate cases where the tip collides with the labware by monitoring it during the z movement in addition to the plunger movement. 
Should help diagnose:
RABR-319
RABR-305
RABR-304

# Test Plan

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

File: api/src/opentrons/hardware_control/backends/ot3controller.py [hunk 1]
```diff
@@ -667,7 +667,8 @@ async def move(
         mounts_moving = [
             k
             for k in moving_axes_in_move_group(move_group)
-            if k in [NodeId.pipette_left, NodeId.pipette_right]
+            if k
+            in [NodeId.pipette_left, NodeId.pipette_right, NodeId.head_l, NodeId.head_r]
         ]
         async with self._monitor_overpressure(mounts_moving):
```
Comment (sfoster1) [inline api/src/opentrons/hardware_control/backends/ot3controller.py]: i don't think this can handle mount axis nodes

**Review by sfoster1 — CHANGES_REQUESTED** [review]

File: api/src/opentrons/hardware_control/backends/ot3controller.py (hunk 1 above)
Comment (ryanthecoder) [reply api/src/opentrons/hardware_control/backends/ot3controller.py]: Yup, added a new util function and fixed that

File: api/src/opentrons/hardware_control/backends/ot3controller.py [hunk 2]
```diff
@@ -664,11 +664,8 @@ async def move(
             else False,
         )
 
-        mounts_moving = [
```
Comment (caila-marashaj) [inline api/src/opentrons/hardware_control/backends/ot3controller.py:670]: nitpick here, but we can maybe rename this variable now to `pipettes_to_monitor` or something to that effect

**Review by caila-marashaj — APPROVED** [review]
nice. I like this.

**Review by sfoster1 — APPROVED** [review]
Okay, but let's add tests at least for `moving_pipettes_in_move_group` please

**Pull request merged**